src/styles/components/charts/styles.less

```diff
@@ -1,342 +1,580 @@
+  }
+
+  .dygraph-chart-wrapper {
+
+    .dot {
+      display: inline-block;
+      margin-right: 6px;
     }
 
-    .text-muted {
-      fill: @body-text-mute-color;
+    .dygraph-label,
+    .graph-legend {
+      font-size: 1rem;
     }
-  }
 
-}
+    .dygraph-axis-label {
+      color: color-lighten(@neutral, 40);
+      font-size: 0.75rem;
 
-.dropdown-chart-details {
+      &.dygraph-axis-label-y {
+        padding-right: 5px;
 
-  position: relative;
-  z-index: @z-index-dropdown-chart-details;
+        em {
+          color: color-lighten(@neutral, 20);
+        }
 
-}
+      }
 
-.dygraph-chart-wrapper {
+    }
 
-  .dot {
+    .graph-legend {
+      display: flex;
+      justify-content: center;
+      margin-top: 10px;
 
-    display: inline-block;
-    margin-right: 6px;
+      & > span {
+        margin-left: 10px;
 
-  }
+        &.disabled {
+          opacity: 0.7;
+        }
+
+        &:hover {
+          color: color-lighten(@black, 40);
+        }
+
+        &:first {
+          margin-left: 0px;
+        }
 
-  .dygraph-label,
-  .graph-legend {
+      }
 
-    font-size: @body-text-font-size;
+    }
 
   }
 
-  .dygraph-axis-label {
+  .dygraph-chart {
+    position: relative;
+    z-index: @z-index-dygraph-chart;
+
+    .dygraph-hover-label {
+      background-color: #171A22;
+      border: 1px solid #000;
+      border-radius: 4px;
+      color: #9B9FA8;
+      font-size: 0.9rem;
+      padding: 6px 8px;
+      pointer-events: none;
+      position: absolute;
+      width: 200px;
+      z-index: @z-index-dygraph-hover-label;
+
+      .highlight {
+        background-color: transparent;
+      }
+
+      span {
+        color: #fff;
 
-    color: color-lighten(@neutral, 40);
-    font-size: @body-text-font-size * 0.75;
+        b {
 
-    &.dygraph-axis-label-y {
+          span {
 
-      padding-right: 5px;
+            &:before {
+              background-color: currentColor;
+              border-radius: 8px;
+              content: '';
+              display: inline-block;
+              height: 8px;
+              margin-right: 4px;
+              width: 8px;
+            }
 
-      em {
+          }
 
-        color: color-lighten(@neutral, 20);
+        }
 
       }
 
     }
 
   }
 
-  .graph-legend {
+  .chart-dialchart {
+    position: absolute;
+
+    .description {
+      display: flex;
+      height: 100%;
+      flex-direction: column;
+      justify-content: center;
+      left: 0;
+      position: absolute;
+      text-align: center;
+      top: 0;
+      width: 100%;
+
+      .icon {
+        display: inline-block;
+      }
+
+    }
+
+  }
 
+  /**********************************/
+  /*  Specific to Nodes Grid View   */
+  /**********************************/
+  .nodes-grid-dials {
     display: flex;
-    justify-content: center;
-    margin-top: 10px;
+    flex-flow: row wrap;
+    justify-content: space-around;
+    margin-left: -(ceil(@base-spacing-unit * 0.3333));
```

Comment:
All these `* 0.3333` should probably be replaced with something like `* (1/3)` or `/ 3` to avoid the 4 point decimal cutoff and let the compiler do the exact math.